tests/testvectors/votetx_tests.json

```diff
@@ -0,0 +1,116 @@
+[
+  {
+    "transaction": {
+      "kind": "string:bns/vote",
+      "chainId": "string:iov-exchangenet",
+      "proposalId": 70,
+      "selection": 0,
```

Comment:
This JSON representation of 

```ts
export enum VoteOption {
  Yes,
  No,
  Abstain,
}
```

can be misleading. There is no guarantee on the raw value/order of those enum cases. A `selection: VoteOption` was never intended to be converted to JSON. I suggest updating IOV Core to

```ts
/**
 * Raw values are used for the JSON representation (e.g. in the wallet to browser extension communication)
 * and must remain unchanged across different semver compatible versions of IOV Core.
 */
export enum VoteOption {
  Yes = "yes",
  No = "no",
  Abstain = "abstain",
}
```

Reply:
Thy will be done.  Thank-you.